### ISSUE TITLE
PREPERROR: There is no package called 'remotes'

### ISSUE THREAD

**Issue by jfiksel** [r-hub/rhub#388]
When using rhub for checking whether my package works, I get the following message:

```
> remotes::install_local("codalm_0.1.0.tar.gz", dependencies = TRUE, INSTALL_opts = "--build")

> Error in loadNamespace(name) : there is no package called ‘remotes’

> Calls: :: ... loadNamespace -> withRestarts -> withOneRestart -> doWithOneRestart

> Execution halted
```

This only happens with the platforms: Fedora Linux, R-devel, clang, gfortran and Ubuntu Linux 16.04 LTS, R-release, GCC, but not Windows. An example of the full log is here: https://builder.r-hub.io/status/codalm_0.1.0.tar.gz-638cc8b935894ce1b96b1d8d12e75696#L254

**Comment by JR-EvolEcol** [comment]
Did you get a response to this?

**Comment by gaborcsardi** [comment]
I think this was a temporary error, please resubmit your job.

**Comment by JR-EvolEcol** [comment]
Dear Gábor,
I still get the preperror on Ubuntu. Please check this link:

https://builder.r-hub.io/status/MHCtools_1.3.0.tar.gz-67a76f9d3eea47ccb05fec9f5bbf6ebe

All the best,
Jacob Roved
PhD, Dept. of Biology
Lund University

On Sep 4, 2020, at 9:23 AM, Gábor Csárdi <notifications@github.com<mailto:notifications@github.com>> wrote:



I think this was a temporary error, please resubmit your job.

**Comment by JR-EvolEcol** [comment]
I also get a note about future file timestamps in Windows:


* checking for future file timestamps ... NOTE
unable to verify current time

Is this something I need to worry about?

All the best,
Jacob Roved
PhD, Dept. of Biology
Lund University


On Sep 4, 2020, at 9:23 AM, Gábor Csárdi <notifications@github.com<mailto:notifications@github.com>> wrote:



I think this was a temporary error, please resubmit your job.

**Comment by gaborcsardi** [comment]
Because one of your dependencies, XML is not available for R 3.6.x, which
is the R version that builder uses.

Re the NOTE:
https://www.mail-archive.com/r-package-devel@r-project.org/msg05866.html

G.
